commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/builders/FileBuilder.kt

```diff
@@ -0,0 +1,14 @@
+package ebi.ac.uk.model.builders
+
+import ebi.ac.uk.model.Attribute
+import ebi.ac.uk.model.File
+
+class FileBuilder {
+    var path: String? = null
+    var size: Long = 0

```

Comment:
we should not populate size with the value provided by the user, this need to be calculated based on file size

Reply:
Done